org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java

```diff
@@ -61,7 +61,9 @@ private Messages() {
   public static String viewInitializing;
 
   public static String actionDelete;
+  public static String actionUnbind;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)